desktop/cmp/form/FormField.scss

```diff
@@ -0,0 +1,38 @@
+.xh-form-field {
+  .bp3-form-content {
+    position: relative;
```

Comment:
Why do we need this, as well as the `position: absolute` below? 

Would presumably be obvious if I was running and removed it, but I am always on the lookout for places where we are customizing the BP classes in non-obvious ways. (Non-obvious if you are not super great at CSS. Colors, font-sizes I think we all grok pretty well and seem less likely to break as BP updates their implementations.)